solidity/random-beacon/test/RandomBeacon.GroupCreation.test.ts

```diff
@@ -1076,7 +1080,9 @@ describe("RandomBeacon - Group Creation", () => {
 
               expect(storedGroup.groupPubKey).to.be.equal(groupPublicKey)
               expect(storedGroup.activationBlockNumber).to.be.equal(0)
-              expect(storedGroup.members).to.be.deep.equal(dkgResult.members)
+              expect(storedGroup.membersHash).to.be.equal(
+                hashUint32Array(dkgResult.members)
+              )
 
               await restoreSnapshot()
             })
```

Comment:
Yes, we can challenge the dkg result, except if we pass a non-ascending order it won't be accepted here. It will revert with `array out of bounds`. This code works and assumes the [ascending order](https://github.com/keep-network/keep-core/blob/main/solidity/random-beacon/contracts/libraries/DKG.sol#L66). 
I've added a test that verifies `array out of bounds` msg. Dkg challange for non-ordered misbehaved array wasn't touched in this PR and there are already tests that cover it.